src/styles/sidebar/components/StyledText.scss

```diff
@@ -99,5 +99,25 @@
     blockquote {
       @apply border-l-[3px] italic px-[1em] text-color-text-light;
     }
+
+    // Un-processed mentions look like plain text
+    a[data-hyp-mention] {
+      @apply text-inherit no-underline;
+    }
+
+    // Valid processed mention with link
+    a[data-hyp-mention][data-hyp-mention-type='link'] {
```

Comment:
We could simplify some of these selectors by assuming that if an element has the `data-hyp-mention-type` attribute then it also has `data-hyp-mention`. Alternatively you could use an attribute just to denote the fact that the mention has been processed, and then use the other attributes to determine the type.

Reply:
> > We could simplify some of these selectors by assuming that if an element has the `data-hyp-mention-type` attribute then it also has `data-hyp-mention`
> 
> I think I found a CSS specificity issue with that, but I slightly changed the approach a couple times, so maybe it does in fact work. Let me try again.

Ok, yeah, it works